internal/status/conditions.go

```diff
@@ -38,6 +39,14 @@ const ReasonValid RouteReasonType = "Valid"
 const ReasonErrorsExist RouteReasonType = "ErrorsExist"
 const ReasonGatewayAllowMismatch RouteReasonType = "GatewayAllowMismatch"
 
+type GatewayClassReasonType string
+
+const reasonValidGatewayClass = "Valid"
+const reasonInvalidGatewayClass = "Invalid"
+
+// clock is used to set lastTransitionTime on status conditions.
+var clock utilclock.Clock = utilclock.RealClock{}
```

Comment:
Ahh we missed this on Conditions! =)

Reply:
Ack. I updated the existing time pkg references so this pkg does not need both "time" and "k8s.io/apimachinery/pkg/util/clock" imports.